app/models/rule_group.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# OpenSCAP RuleGroup
+class RuleGroup < ApplicationRecord
+  # Need to set primary key format to work with uuid primary key column
+  has_ancestry(primary_key_format: %r{\A[\w\-]+(\/[\w\-]+)*\z})
+
+  has_many :profile_rule_groups, dependent: :delete_all
+  has_many :profiles, through: :profile_rule_groups, source: :profile
+  has_many :left_rule_group_relationships, dependent: :delete_all, foreign_key: :left_id,
+                                           inverse_of: :left, class_name: 'RuleGroupRelationship'
+  has_many :right_rule_group_relationships, dependent: :delete_all, foreign_key: :left_id,
+                                            inverse_of: :left, class_name: 'RuleGroupRelationship'
```

Comment:
@marleystipich2 also here

```suggestion
                                            inverse_of: :right, class_name: 'RuleGroupRelationship'
```